src/mol-gl/shader/postprocessing.frag.ts

```diff
@@ -169,13 +169,25 @@ void main(void) {
             float viewDist = abs(getViewZ(closestTexel));
             float fogFactor = smoothstep(uFogNear, uFogFar, viewDist);
             if (!uTransparentBackground) {
+                if (isTransparentOutline > 0.0) {
+                    float outlineAlpha = clamp(isTransparentOutline * 2.0, 0.0, 1.0);
+                    color.rgb = mix(color.rgb, mix(uOutlineColor, uFogColor, fogFactor), outlineAlpha);
+                } else {
                     color.rgb = mix(uOutlineColor, uFogColor, fogFactor);
+                }                    
             } else {
-                color.a = 1.0 - fogFactor;
-                color.rgb = mix(uOutlineColor, vec3(0.0), fogFactor);
+                if (isTransparentOutline > 0.0) {
+                    float outlineAlpha = clamp(isTransparentOutline * 2.0, 0.0, 1.0);
+                    vec3 outlineColor = mix(uOutlineColor, uFogColor, fogFactor);                    
+                    color.a = color.a + outlineAlpha * (1.0 - color.a); 
+                    color.rgb = mix(color.rgb, outlineColor, outlineAlpha);
+                } else {
+                    color.a = 1.0 - fogFactor;
+                    color.rgb = mix(uOutlineColor, vec3(0.0), fogFactor);
+                }                
             }
             #ifdef dBlendTransparency
```

Comment:
I would propose to move all the #ifdef dBlendTransparancy block before the outline. It avoid blending the outline with the background when opactity are low, if you try opacity 0.04, you will actually see a background color outline...
before 
<img width="764" height="628" alt="image" src="https://github.com/user-attachments/assets/c2b7fffa-519a-46f9-ae3c-b206899e6dad" />
after
<img width="864" height="594" alt="image" src="https://github.com/user-attachments/assets/7c617dbb-5b62-48fa-a0b0-f4e59e703cdd" />